exploratory_analysis.ipynb

```diff
@@ -1757,125 +1758,136 @@
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "    state                     measure_name  state_code  county_code  \\\n",
-       "477    TX  Sexually transmitted infections     35616.0     188830.0   \n",
-       "115    GA  Sexually transmitted infections      6188.0      76906.0   \n",
-       "279    MS  Sexually transmitted infections      6860.0      20077.0   \n",
-       "499    VA  Sexually transmitted infections     20349.0     106081.0   \n",
-       "301    NC  Sexually transmitted infections     11100.0      30000.0   \n",
+       "                        measure_name state  county  state_code  county_code  \\\n",
+       "94               Children in poverty    TX     762         762          762   \n",
+       "142    Daily fine particulate matter    TX     762         762          762   \n",
+       "448                     Unemployment    TX     762         762          762   \n",
+       "397  Sexually transmitted infections    TX     742         742          742   \n",
+       "193               Diabetic screening    TX     741         741          741   \n",
        "\n",
-       "     measure_id  numerator  denominator  raw_value   fips_code  start_year  \n",
-       "477     33390.0   350383.0   75100757.0   237403.1  35804830.0   1491400.0  \n",
-       "115     21420.0   137386.0   29185253.0   207325.2   6264906.0    956759.0  \n",
-       "279     11025.0    65244.0    8847819.0   191435.0   6880077.0    492449.0  \n",
-       "499     17955.0    98006.0   23841006.0   157276.4  20455081.0    801988.0  \n",
-       "301     13500.0   137905.0   28414196.0   119505.5  11130000.0    603000.0  "
+       "     measure_id  numerator  denominator  raw_value  fips_code  start_year  \n",
+       "94          762        762            0        762        762         762  \n",
+       "142         762          0            0        762        762         762  \n",
+       "448         762        762          762        762        762         762  \n",
+       "397         742        742          731        742        742         742  \n",
+       "193         741          0          741        741        741         741  "
       ]
      },
-     "execution_count": 225,
+     "execution_count": 255,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# Top five states with the higest raw_value in group 1\n",
+    "# Top five measures with the most occurance in different states in group 1\n",
     "\n",
-    "df1_state = df1.groupby(['state', 'measure_name']).sum().reset_index()\n",
-    "df1_state.sort_values('raw_value', ascending=False).head()"
+    "df1_state = df1.groupby(['measure_name','state']).count().reset_index()\n",
+    "d1 = df1_state.sort_values('raw_value', ascending=False).head()\n",
```

Comment:
Did you want this in a new d f named "d1" or was this supposed to make a new copy of df1_state?

Reply:
Never mind, I see what you did here